__tests__/mockResponses.ts

```diff
@@ -35,10 +35,6 @@ export const successfulComparisonResponse = {
         'https://github.com/octocat/Hello-World/commit/f0c556fd81d38b23ff2b9c7c9d192ff426ff35bbc4',
       commit: {
         message: 'Title of my commit\nBody of my message'
-      },
```

Comment:
the duplicate case is already tested with `octocat` so i removed this one to test the no author case